databases/sql-1.md

```diff
@@ -57,7 +57,7 @@ Most software applications store data of some sort -- user data, content, etc. D
 
 1. PostgreSQL is a popular, open-source version of SQL. Try it out:
    - In Terminal, type `psql` to use PostgreSQL.
-   - Try adding a table with the command `CREATE TABLE food (name TEXT, calories INTEGER);`
+   - Try adding a table with the command `CREATE TABLE food (id INTEGER,name TEXT, calories INTEGER);`
```

Comment:
Could you include a space after the comma here to be consistent?

Reply:
Hi @lalpert  thank you for your comment I added the comma and spacing as well as fixed the conflict which was that I added `\dt` to display table.